understanding/understanding-techniques.html

```diff
@@ -229,150 +209,134 @@ <h3>Numbered Lists, "AND"</h3>
                		
                <section id="advisory-techniques">
                   			
-                  <h2>Advisory Techniques</h2>
+                  <h2>参考テクニック</h2>
                   			
-                  <p><em>Advisory techniques</em> are suggested ways to improve accessibility. They are often very helpful to some
-                     users, and may be the only way that some users can access some types of content.
+                  <p><em>参考テクニック</em>はアクセシビリティを改善するために推奨される方法である。これは一部の利用者にとっては非常に有用であり、またある利用者がある種のコンテンツにアクセスできる唯一の方法となることもあるかもしれない。
                   </p>
                   			
-                  <p>Advisory techniques are not designated as sufficient techniques for various reasons
-                     such as:
+                  <p>参考テクニックは以下のようなさまざまな理由で、十分なテクニックとはならない:
                   </p>
                   			
                   <ul>
                      				
                      <li>
                         					
-                        <p>they may not be sufficient to meet the full requirements of the success criteria;</p>
+                        <p>達成基準のすべての要求事項を満たすのに十分でない。</p>
                         				
                      </li>
                      				
                      <li>
                         					
-                        <p>they may be based on technology that is not yet stable;</p>
+                        <p>まだ安定していない技術に基づいている。</p>
                         				
                      </li>
                      				
                      <li>
                         					
-                        <p>they may not be <a href="conformance#accessibility-support">accessibility supported</a> in many cases (for example, assistive technologies do not work with them yet);
+                        <p>多くの場合において<a href="conformance#accessibility-support">アクセシビリティ サポーテッド</a>でない (例えば、支援技術がテクニックに対応できていない)。
```

Comment:
括弧内ですが yet の訳が落ちているように思います。また them を「テクニック」と補っていますが、これは補わなくても通じるのではないでしょうか。というところでこんなふうにしてはどうでしょうか。

```suggestion
                        <p>多くの場合において<a href="conformance#accessibility-support">アクセシビリティ サポーテッド</a>でない (例えば、支援技術がまだ対応できていない)。
```

Reply:
取り込みました。